pandas-stubs/core/groupby/generic.pyi

```diff
@@ -208,26 +209,35 @@ class SeriesGroupBy(GroupBy[Series[S2]], Generic[S2, ByT]):
 
 _TT = TypeVar("_TT", bound=Literal[True, False])
 
+class DFCallable1(Protocol):
+    def __call__(self, df: DataFrame, /, *args, **kwargs) -> Scalar | list | dict: ...
```

Comment:
Could also add the `include_groups` keyword  https://pandas.pydata.org/docs/reference/api/pandas.core.groupby.DataFrameGroupBy.apply.html

Reply:
(Weird deprecation warning: " Setting include_groups to True is deprecated. Only the value False will be allowed in a future version of pandas.")

Reply:
Yes, I think this is a temporary parameter to handle the changeover in behavior, so I don't think we should put it in the stubs.